src/classes/FORM_ServiceGiftEntry_TEST.cls

```diff
@@ -42,38 +42,34 @@ public with sharing class FORM_ServiceGiftEntry_TEST {
     */
     @IsTest
     static void shouldReturnCorrectNumberOfRecords() {
-        FORM_Service fs = new FORM_Service();
-
         FORM_Template template0 = UTIL_UnitTestData_TEST.createSampleTemplate();
         String templateJSON0 = JSON.serialize(template0);
-        fs.storeFormTemplate(null,template0.name, template0.description, template0.version, templateJSON0);
+        FORM_ServiceGiftEntry.storeFormTemplate(null,template0.name, template0.description, template0.version, templateJSON0);
```

Comment:
I notice we are moving away from instantiating the FORM_Service class in favor of calling static methods here - we frequently talk about wanting to shift the other direction, towards instances of classes that can be mocked and behaviors  can be controlled by the caller. Just want to make sure we are aware of and ok with the switch in the opposite directly here!